### PR TITLE
Time: Use moment.js

### PR DESCRIPTION
**Pull request by MrChrisW**
Time use moment.js. 

Addressing #1554 

Investigating #2372

___________________
https://duck.co/ia/view/time

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@MrChrisW Thanks! FWIW, I had this before: https://github.com/duckduckgo/zeroclickinfo-spice/pull/1795/files -- not sure if it's helpful

**Comment by jagtalon** [comment]
@MrChrisW sorry for the delay on this--checking it out!

File: share/spice/time/time.js [hunk 1]
```diff
@@ -69,56 +69,45 @@
...
-        //Convert 24 hour time to 12 hour time
-        function toPrettyTime(date) {
-            var hours = date.getHours(),
-                minutes = date.getMinutes();
-            return {
-                hours: (hours % 12) || 12,
-                minutes: minutes < 10 ? '0' + minutes : minutes,
-                amPM: hours >= 12 ? "PM" : "AM"
+            var dateTime = {
+                time: dateObj.format("LT"),
```
Comment (MrChrisW) [inline share/spice/time/time.js]: @jagtalon the use of "LT" is not needed here. See [comment](https://github.com/duckduckgo/zeroclickinfo-spice/issues/2372#issuecomment-170736182) for details.

**Comment by jagtalon** [comment]
@MrChrisW it looks like I'm getting the wrong day of the month:

screen shot 2016-02-22 at 9 48 38 am

Are you seeing the same thing?

**Comment by MrChrisW** [comment]
@jagtalon :+1:  Try that. cfb1ed6

**Comment by jagtalon** [comment]
:+1: yass

**Pull request merged**